src/sections/offering/hedvig-info.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { colors } from '@hedviginsurance/brand'
+import { Header } from './price-info'
+import { Container, InnerContainer, Card } from './insured-amount'
+
+interface Props {
+  title: string
+}
+
+export const Row = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'baseline',
+  '@media (max-width: 710px)': {
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+})
+
+export const Col = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+  marginBottom: '30px',
+})
+
+const Image = styled('img')({
+  margin: '20px',
+  maxWidth: '210px',
+  '@media (max-width: 710px)': {
+    maxWidth: '300px',
+  },
+  '@media (max-width: 400px)': {
+    maxWidth: '210px',
+  },
+})
+
+export const Title = styled('label')({
+  marginBottom: '10px',
+  fontSize: '16px',
+  textAlign: 'center',
+  color: colors.BLACK,
+  maxWidth: '210px',
+})
+
+export const SubTitle = styled('label')({
+  fontSize: '16px',
+  textAlign: 'center',
+  color: colors.DARK_GRAY,
+  maxWidth: '210px',
+})
+
+export const HedvigInfo: React.SFC<Props> = (props) => (
+  <Container>
+    <InnerContainer>
+      <Card>
+        <Header>{props.title}</Header>
+        <Row>
+          {COLUMNS.map((col, index) => (
+            <Col key={index}>
```

Comment:
generally its preferred not to use the index as the key, as it may change independently of the item if we i.e. put another order or remove an item from the array. I think we should add a `key` prop to each item in the `COLUMNS` array